modules/restore-add-master.adoc

```diff
@@ -0,0 +1,101 @@
+ETCD_INITIAL_ADVERTISE_PEER_URLS="https://etcd-1.clustername.devcluster.openshift.com:2380"
+ETCD_INITIAL_CLUSTER_STATE="existing"'
+Member  1e42c7070decd39 added to cluster 6e25bab1bb556673
+Starting etcd..
+----
+
+. Verify that the new member is in the list of Pods associated with etcd and is actively running:
++
+----
+[core@ip-10-0-147-172 ~]$ oc get pods -n openshift-etcd
+NAME                                                     READY   STATUS    RESTARTS   AGE
+etcd-member-ip-10-0-128-73.us-east-2.compute.internal    2/2     Running   0          15h
+etcd-member-ip-10-0-147-172.us-east-2.compute.internal   2/2     Running   7          122m
+etcd-member-ip-10-0-171-108.us-east-2.compute.internal   2/2     Running   0          15h
+----
+
+. Verify that the etcd member has been successfully added to the etcd cluster:
+
+.. Connect to the running etcd container:
++
+----
+[core@ip-10-0-147-172 ~] id=$(sudo crictl ps --name etcd-member | awk 'FNR==2{ print $1}') && sudo crictl exec -it $id /bin/sh
+----
+
+.. In the etcd container, export variables needed for connecting to etcd:
++
+----
+sh-4.2# export ETCDCTL_API=3 ETCDCTL_CACERT=/etc/ssl/etcd/ca.crt ETCDCTL_CERT=$(find /etc/ssl/ -name *peer*crt) ETCDCTL_KEY=$(find /etc/ssl/ -name *peer*key)
+----
+
+.. In the etcd container, execute `etcdctl member list` and verify that the new member is listed:
++
+----
+sh-4.2#  etcdctl member list -w table
+
++------------------+---------+------------------------------------------+------------------------------------------------------------------+---------------------------+
+|        ID        | STATUS  |                   NAME                   |                            PEER ADDRS                            |       CLIENT ADDRS        |
++------------------+---------+------------------------------------------+------------------------------------------------------------------+---------------------------+
+| 29e461db6be4eaaa | started | etcd-member-ip-10-0-128-73.us-east-2.compute.internal | https://etcd-2.clustername.devcluster.openshift.com:2380 | https://10.0.128.73:2379 |
+|  cbe982c74cbb42f | started | etcd-member-ip-10-0-147-172.us-east-2.compute.internal | https://etcd-0.clustername.devcluster.openshift.com:2380 | https://10.0.147.172:2379 |
+| a752f80bcb0da3e8 | started |   etcd-member-ip-10-0-171-108.us-east-2.compute.internal | https://etcd-1.clustername.devcluster.openshift.com:2380 |   https://10.0.171.108:2379 |
++------------------+---------+------------------------------------------+------------------------------------------------------------------+---------------------------+
+----
++
+Note that it may take up to 10 minutes for the new member to start.
```

Comment:
Should this be in a note block? I can only find one other instance in 4.x docs where we use "note that" in paragraph.

Reply:
I actually do think we use this a decent bit, but I think in this case I can remove the "Note that" and just start with "It".